mycli/myclirc

```diff
@@ -43,6 +43,12 @@ table_format = ascii
 # Screenshots at http://mycli.net/syntax
 syntax_style = default
 
+# Keybindings:
+# When Vi mode is enabled you can use modal editing features offered by Vi in the REPL.
+# When Vi mode is disabled emacs keybindings such as Ctrl-A for home and Ctrl-E
+# for end are available in the REPL.
+vi = False
```

Comment:
Should we use the existing `myclirc` option for this? The one below on line 55.

Reply:
Ah yes, I've searched for vi in myclirc and didn't found anything, it was called emacs :smile:. I've restored/fixed `mycli/clitoolbar.py`.